inc/conditionnabletrait.class.php

```diff
@@ -76,18 +76,16 @@ public function checkConditions(array $input): bool {
       return true;
    }
 
-   public function updateConditions($input) : bool {
+   public function checkConditionSettings(array $input): bool {
```

Comment:
data integrity checks for conditions are now passive : no more change in the DB. IT is the responsibility of the itemtype to set show_rule accordying to the data integrity check in prepareInputForUpdate().